templates/library.html

```diff
@@ -7,6 +7,13 @@
         <div class="col-md-8">
 
     <h1 style="margin-top: 00px;" id="libraryName">{{library.name}}</h1>
+    {{#watch}}{{#star}}{{#fork}}
```

Comment:
I think we can just check one of them, since they'll always be there together, right?